### DISCUSSION TITLE
Where can I find about changes introduced on v11 release candidate versions?

### DISCUSSION THREAD

**Discussion by cabello** [trpc/trpc#6235] [Q&A] [answered]
Thank you for the trpc project, it has been lovely to build apps with this.

We use v11 in production and from time to time we update to the latest rc.

We can see the releases in https://www.npmjs.com/package/@trpc/server?activeTab=versions
I see an active `next` branch: https://github.com/trpc/trpc/commits/next/
It's unclear what changes happened between two releases, for example `11.0.0-rc.632` and `11.0.0-rc.633`.

Thank you again 🙇

**Comment by KATT** [comment, marked as answer]
https://trpc.io/docs/migrate-from-v10-to-v11 is where you find a changelog

We don't have any breaking changes planned for 11.0.0, we're ripping the bandaid off soon.

**Reply by cabello** [reply]
I used this guide for the migration and noticed a commit here and there adding to it. I was interested on being able to follow the changes that are happening between each release candidate.
